CONTRIBUTING.md

```diff
@@ -201,8 +201,7 @@ implementation files('libs/ably-android-1.2.42.aar')
 This library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
 
 1. Create a branch for the release, named like `release/1.2.4` (where `1.2.4` is what you're releasing, being the new version)
-2. Replace all references of the current version number with the new version number (check the [README.md](./README.md) and [common.gradle](./common.gradle)) and commit the changes
-    a. Increment the `versionCode` in the Android project's `build.gradle` by 1
```

Comment:
Can we just double check if updated steps work for release process ?

Reply:
yes, I double checked this, now we can even create github workflow to make a release for us